pkg/api/validation/validation.go

```diff
@@ -58,6 +58,8 @@ var DNS1123LabelErrorMsg string = fmt.Sprintf(`must be a DNS label (at most %d c
 var DNS952LabelErrorMsg string = fmt.Sprintf(`must be a DNS 952 label (at most %d characters, matching regex %s): e.g. "my-name"`, validation.DNS952LabelMaxLength, validation.DNS952LabelFmt)
 var pdPartitionErrorMsg string = IntervalErrorMsg(0, 255)
 var PortRangeErrorMsg string = IntervalErrorMsg(0, 65536)
+var GroupIdRangeErrorMsg string = IntervalErrorMsg(-1, 2147483648)
```

Comment:
@thockin @pmorie I got this limit from the error tip when create pod with invalid value, 
"Failed to start with docker id bcf625727065 with error: API error (500): Cannot start container bcf6257270654a05aa6598330451919a63bb325ce95a531bb0ddadc0f9633ee2: [8] System error: Uids and gids must be in range 0-2147483647" refer the issue #16352  I linked.
They must be unsigned value and grater than 0 and less than 2147483648.  (can equal 0 and 2147483647, so I set the error tip to -1, 2147483648)